schema/Publish.sql

```diff
@@ -0,0 +1,310 @@
+﻿/*
```

Comment:
How did you generate this code?

Reply:
From visual studio 2019 SQL Server Object Explorer ,we can use the the below steps to replicate for a database

- Extract Data-tier Application to generate the SQL schema and save (extension will be .dacpac) to local
 - Right Click Databases, select Publish Data-tier Application option 
 - Browse and choose SQL Schema file(.dacpac) file, in local, generated in first step and choose generate Script.